File: src/main/java/br/com/newObjects/model/classes/NovoFornecedor.java
package br.com.newObjects.model.classes;

import br.com.model.entities.classes.Usuario.Fornecedor;

public class NovoFornecedor {

    public static Fornecedor novoFornecedor() {
        Fornecedor fornecedor = new Fornecedor("TesteLTDA", "Empresa de teste", "000.000.000-00", "senhaTeste",
                "email@testeFornecedor.com.br");

        NovoEndereco.novaListaEnderecoFornecedor(fornecedor);

        fornecedor.setListaTelefones(NovoTelefone.novaListaTelefoneFornecedor(fornecedor));
        return fornecedor;
    }
}

File: src/test/java/br/com/newObjects/model/usuario/NovoFornecedor.kt
package br.com.newObjects.model.usuario

import br.com.model.usuario.Fornecedor
import br.com.newObjects.model.NovoEndereco.novaListaEndereco
import br.com.newObjects.model.NovoTelefone.novaListaTelefones

object NovoFornecedor {

    private val usuario = NovoUsuario.novoUsuario()

    fun novoFornecedor(): Fornecedor {
        val fornecedor = Fornecedor(
            usuario.nome, usuario.cnpjCpf, usuario.senha, usuario.email
        )
        novaListaEndereco(fornecedor)
        novaListaTelefones(fornecedor)
        return fornecedor
    }
}
